src/Test.EndToEnd.Baselining/BaseliningDetailLogger.cs

```diff
@@ -89,7 +97,12 @@ private void Write(char marker, string guid)
 
         public void Dispose()
         {
-            if (Writer != null)
+            Dispose(true);
+        }
+
+        protected virtual void Dispose(bool disposing)
```

Comment:
>protected [](start = 8, length = 9)

You can add the DIspose(true) pattern if you want but if you don't have native interop, it's not strictly necessary. This pattern exists for finalized objects that weren't disposed by callers but have an urgent need to make sure native handles are disconnected properly.

Reply:
Not necessary but the IDE now squiggles all classes that don't have both overloads. I want all the squiggles I see to really be errors worth fixing, so I've been converting to this form to "fix" these warnings from VS.